contracts/VotingMachines/AbsoluteVote.sol

```diff
@@ -31,7 +31,7 @@ contract AbsoluteVote is IntVoteInterface {
         bool open; // voting open flag
     }
 
-    event VoteProposal(bytes32 indexed _proposalId, address indexed _avatar, address indexed _voter, uint _vote, uint _reputation, bool _isOwnerVote);
+    event AVVoteProposal(bytes32 indexed _proposalId, address indexed _avatar, address indexed _voter, uint _vote, uint _reputation, bool _isOwnerVote);
```

Comment:
the AVVoteProposal should include only the extended params which are not included on VoteProposal interface (with the proposalId as Index).

Reply:
Removed